Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM python:3.5
-WORKDIR /code
-ADD . /code
```

Comment:
Switching back to master is a feature. It allows to test the docker integration before making a tag.